rasr/command.py

```diff
@@ -151,7 +163,8 @@ def run_script(
                     print(
                         "'%s' crashed - copy temporary work folder as 'crash_dir'" % cmd
                     )
-                    shutil.copytree(tmp_dir, "crash_dir_" + date_time_cur)
+                    if "crash_dir_%d" % task_id not in os.listdir(work_dir):
```

Comment:
Only copy when it is not in the work dir seems a bit risky. I would prefer the version with the task id and the datetime.